src/eddymotion/model/base.py

```diff
@@ -448,6 +449,69 @@ class DKIModel(BaseModel):
     _model_class = "dipy.reconst.dki.DiffusionKurtosisModel"
 
 
+class GaussianProcessModel:
+    """A Gaussian process model for DWI data based on [Andersson15]_."""
+
+    __slots__ = (
+        "_dwi",
+        "_kernel",
+        "_num_iterations",
```

Comment:
What is this slot for? How can it be used at the moment?

Reply:
Removed in 10b4a6b. Sorry.